airflow/providers/qubole/operators/qubole.py

```diff
@@ -250,9 +250,9 @@ def on_kill(self, ti=None) -> None:
         else:
             self.get_hook().kill(ti)
 
-    def get_results(self, ti=None, fp=None, inline: bool = True, delim=None, fetch: bool = True) -> str:
+    def get_results(self, ti=None, fp=None, inline: bool = True, delim=None, fetch: bool = True, include_headers: bool = False) -> str:
         """get_results from Qubole"""
-        return self.get_hook().get_results(ti, fp, inline, delim, fetch)
+        return self.get_hook().get_results(ti, fp, inline, delim, fetch,include_headers)
```

Comment:
```suggestion
        return self.get_hook().get_results(ti, fp, inline, delim, fetch, include_headers)
```

Before pylint wastes a CI run to catch it…

Reply:
Thanks, not sure why I missed it. 
Fixed along with some other warnings